project/plugins.sbt

```diff
@@ -9,4 +9,6 @@ addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.5.2")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.12")
 
+// addSbtPlugin("com.gradle" % "sbt-develocity" % "1.0.1")
```

Comment:
I used develocity to debug the build and find optimization opportunities. We should eventually enable it in CI.